modules/sugar/src/test/ts/browser/BodyTest.ts

```diff
@@ -16,12 +17,65 @@ UnitTest.test('BodyTest', () => {
   assert.eq(false, Body.inBody(div));
   assert.eq(false, Body.inBody(child));
   assert.eq(false, Body.inBody(text));
+});
 
-  Insert.append(body, div);
-  assert.eq(true, Body.inBody(div));
-  assert.eq(true, Body.inBody(child));
-  assert.eq(true, Body.inBody(text));
-  assert.eq(true, Body.inBody(body));
+UnitTest.test('Body.inBody - elements in body', () => {
+  const body = SelectorFind.first('body').getOrDie();
 
-  Remove.remove(div);
+  const div = Element.fromTag('div');
+  const child = Element.fromTag('span');
+  const text = Element.fromText('hi');
+  Insert.append(child, text);
+  Insert.append(div, child);
+  Insert.append(body, div);
+  try {
```

Comment:
Another place where we maybe shouldn't have a try/finally?

Reply:
Ta. Fixed.